app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,31 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+@import 'home';
+body {
+padding-top: 60px;
```

file này ko đúng indent nhé
```suggestion
  padding-top: 60px;
```
Indent: thụt lề- 1 tab = 2spaces. vào line này tab  vào 1 tab = 2 space so với line 4

self check file này